diff --git a/node_modules/discord.js/src/managers/ApplicationCommandManager.js b/node_modules/discord.js/src/managers/ApplicationCommandManager.js
index 7d08dc9..4e10df4 100644
--- a/node_modules/discord.js/src/managers/ApplicationCommandManager.js
+++ b/node_modules/discord.js/src/managers/ApplicationCommandManager.js
@@ -248,7 +248,7 @@ class ApplicationCommandManager extends CachedManager {
           : command.defaultMemberPermissions;
     }
 
-    return {
+    var retVal = {
       name: command.name,
       name_localizations: command.nameLocalizations ?? command.name_localizations,
       description: command.description,
@@ -259,6 +259,11 @@ class ApplicationCommandManager extends CachedManager {
       default_member_permissions,
       dm_permission: command.dmPermission ?? command.dm_permission,
     };
+
+    if ('integration_types' in command) retVal['integration_types'] = command.integration_types;
+    if ('contexts' in command) retVal['contexts'] = command.contexts;
+
+    return retVal;
   }
 }
 
